h/db/__init__.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=import-outside-toplevel
+# pylint: disable=import-outside-toplevel, cyclic-import
```

Comment:
I don't love not solving this, but I think there are two things going against us:

* `pylint`'s output is actually very unhelpful for resolving this, as it tags the wrong lines and files, leaving you to guess
* `h` is full of spaghetti